package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "cat-me-if-you-can",
+  "name": "cat-project",
```

Comment:
why?

Reply:
I have it named as that on my machine so it looks like it named it that in the package-lock.json. Mistake.

Reply:
hmm. maybe we should set a name in package.json to avoid this